text/chapter3.md

```diff
@@ -560,20 +604,60 @@ apply f x = f x
 infixr 0 apply as $
 ```
 
-So `apply` takes a function and a value and applies the function to the value. The `infixr` keyword is used to define `($)` as an alias for `apply`.
+So `apply` takes a function and a value and applies the function to the value. The `infixr` keyword is used to define `$` as an alias for `apply`.
 
 But why would we want to use `$` instead of regular function application? The reason is that `$` is a right-associative, low precedence operator. This means that `$` allows us to remove sets of parentheses for deeply-nested applications.
 
 For example, the following nested function application, which finds the street in the address of an employee's boss:
 
 ```haskell
-street (address (boss employee))
+_.street (_.address (_.boss employee))
```

Comment:
I think something like:
```purs
book3 = insertEntry john (insertEntry peggy (insertEntry ned emptyBook))

book6 = insertEntry john $ insertEntry peggy $ insertEntry ned emptyBook
```
from the the sequence of examples proposed earlier might be better here.